pkg/mcs/scheduling/server/grpc_service.go

```diff
@@ -254,6 +257,74 @@ func (s *Service) GetOperator(ctx context.Context, request *schedulingpb.GetOper
 	}, nil
 }
 
+// AskBatchSplit implements gRPC PDServer.
+func (s *Service) AskBatchSplit(ctx context.Context, request *schedulingpb.AskBatchSplitRequest) (*schedulingpb.AskBatchSplitResponse, error) {
```

Comment:
It seems to be similar to `HandleAskBatchSplit`, could we reduce it?

Reply:
What do you mean?